### DISCUSSION TITLE
一氧化二氫 - 危險的恐怖化合物

### DISCUSSION THREAD

**Discussion by zhilin93517** [1101-GS4719-C/Python-Community-GS4719-C#2181] [General]
一氧化二氫 DYMO 一種危險的聚合物，全名叫 Dihydrohen monoxide。
這種聚合物在常溫常壓下是一種無色無味的液體，若不小心吸入鼻腔可能導致窒息，每年導致多起死亡事故。此外，若攝取過多的一氧化二氫將會導致排尿過多，下肢腫感、噁心想吐和電解質失衡，在使用時應注意攝取量以避免一氧化二氫中毒。

我曾經在意外中攝入一氧化二氫，當時感受到極度的不舒適，鼻腔的刺痛感延伸到後腦勺，無法控制地強力的咳嗽持續數分鐘，過了良久情況才稍緩，我認為在使用一氧化二氫十不可以開玩笑，要注意使用方式，以免發生類似的事情。

**Comment by BSC111202528** [comment]
真的覺得這種化合物非常危險，上次不小心在一分鐘食用300cc，醫生說我少了60秒的壽命，我認為大家需要認知到這件是的嚴重性，唯有這樣才能預防這樣的情況發生。

**Reply by gigitilto** [reply]
您說的非常有道理，但我想請問，食用量與減少的壽命是成正相關嗎？因為我曾多次食用這個物質，但是我感受起來與不食用沒什麼差別，因此我並不了解這個物質帶來的危害。

**Comment by poyen0806** [comment]
上次跑完步，滿頭大汗的我一把抓起水壺，沒想太多就猛灌了好幾口。正喝得舒暢時，旁邊一個朋友突然一臉嚴肅地跟我說：「你知道那是什麼嗎？一氧化二氫耶！」

我手一抖，差點把水噴出來，腦中響起科學頻道裡警告的聲音：「這化合物可是有名的腐蝕性，可以把石頭、金屬甚至你的皮膚弄壞！而且它在全球每年導致數千人死亡！」我人直接愣住，心跳加速，一邊回想自己剛剛到底喝了多少，一邊強裝鎮定問：「真的嗎？那我...怎麼辦？」

朋友看我表情慌張，突然壓低聲音說：「其實這種化合物還有個名字，叫……水。」

我聽完瞬間石化，還沒從驚嚇中恢復，就又給自己灌了一口水，心裡暗想：「還好，這水還沒到期，先喝為敬。」
